### DISCUSSION TITLE
Floating labels

### DISCUSSION THREAD

**Discussion by viniciusjdebiasi** [twbs/bootstrap#40966] [Q&A] [answered]
Hello everyone! 

I have a little problem and I can't solve it.

I'm using Floating labels and I'm not able to adjust a style.
`<div class="form-floating mb-3">
            <input
              type="email"
              class="form-control input-custom"
              id="floatingInput"
              placeholder="name@example.com"
            />
            <label for="floatingInput" className="label">
              Indirizzo email o numero di cellulare
            </label>
          </div>`
![foto_duvida](https://github.com/user-attachments/assets/c2abd0ab-8e73-44aa-9b96-25968d96a1a1)

**Comment by ynng3** [comment, marked as answer]
As an example, let's assume we want to apply a transparent background to floating labels and write the code accordingly.

Access the element using the `:after` pseudo-selector as shown in the code below.

```scss
.form-floating>.form-control:focus~label::after { background-color: transparent; }
```

**Reply by viniciusjdebiasi** [reply]
Hello! It worked perfectly, thank you very much for your help!
